storage/FileManagerPosix.cpp

```diff
@@ -86,6 +88,7 @@ size_t FileManagerPosix::numSlots(const block_id block) const {
   }
 
   if ((file_stat.st_size % kSlotSizeBytes) != 0) {
+    LOG(ERROR) << "Size of file " << filename.c_str() << " was not integral.";
```

Comment:
As above, this is related to the secondary bug which is mentioned above. I like the suggestion to change to LOG(FATAL)- addressed.